test/unit/src/PolyLineTest.cpp

```diff
@@ -8,89 +8,128 @@ using namespace ci;
 using namespace ci::app;
 using namespace std;
 
-TEST_CASE("PolyLine", "Orientation")
```

Comment:
I didn't really understand how catch worked when I set this up. It seems better now to make Orientation it's own section rather than a top level case.